imap_processing/cdf/config/imap_lo_l1a_variable_attrs.yaml

```diff
@@ -146,7 +147,7 @@ esa_step:
   VALIDMAX: 7
   FORMAT: I1
   CATDESC: Energy Step
-  DEPEND_1: esa_step
+  DEPEND_1: direct_events
```

Comment:
could this be extra now since `de_default` already has it? you may just need to remove it?